docs/release-notes/2.37.0.md

```diff
@@ -0,0 +1,170 @@
+
+## Bugs Fixed
+
+
+- [PMM-8690](https://jira.percona.com/browse/PMM-8690) - Fixed an issue where the Replica set lag displayed incorrect values on the MongoDB ReplSet Summary dashboard for an ARBITER node.
+- [PMM-9775](https://jira.percona.com/browse/PMM-9775) - QAN: PMM failed to display the set timezone for QAN despite it being set to UTC.
+- [PMM-10687](https://jira.percona.com/browse/PMM-10687) - DBaaS: Fixed an issue where DB cluster creation failed when using an unreleased version of pmm-server.
+- [PMM-11121](https://jira.percona.com/browse/PMM-11121) - DBaaS: Fixed an issue where the list of DB clusters did not load if one of the K8s clusters was unresponsive.
+- [PMM-11226](https://jira.percona.com/browse/PMM-11226) - Fixed an issue where different operator versions and names were displayed for each K8s cluster.
+- [PMM-11313](https://jira.percona.com/browse/PMM-11313) - Rows Fetched/Read and Users by Rows Read dashboards are not working in the MySQL User Details dashboard
+- [PMM-11555](https://jira.percona.com/browse/PMM-11555) - For thousands of client connections for ProxySQL, the graph for Active Frontend connections did not render due to sluggish performance. Furthermore, this could affect the rendering and usage of the ProxySQL Instance summary dashboard. The issue has been resolved now.
+- [PMM-11829](https://jira.percona.com/browse/PMM-11829) - PMM Agent does not work with PG 12 and PGSM 2.0. This issue has been resolved now.
+- [PMM-11844](https://jira.percona.com/browse/PMM-11844) - Fixed an issue where a user failed to add PSMDB with pmm-admin using native LDAP authentication and TLS.
+- [PMM-11862](https://jira.percona.com/browse/PMM-11862) - Fixed inconsistent naming of the Advisors feature across PMM UI.
+- [PMM-11875](https://jira.percona.com/browse/PMM-11875) - Fixed an issue where the users could not select and compare multiple nodes on the Summary dashboard.
+- [PMM-11904](https://jira.percona.com/browse/PMM-11904) - Index Size and Concurrent Inserts panels were broken on the Insight / VictoriaMetrics dashboard after updating to version 1.89.1.
+- [PMM-10795](https://jira.percona.com/browse/PMM-10795) - Node summary was not visible under MongoDB ReplSet Summary when the node name contained a dot (.) in the name.
+- [[PMM-11465](https://jira.percona.com/browse/PMM-11465) - QAN: After upgrading PMM to 2.33.0, the EXPLAIN tab displayed an empty output.
+- [PMM-11729](https://jira.percona.com/browse/PMM-11729) - Syntax issues with Placeholders
+- [PMM-11849](https://jira.percona.com/browse/PMM-11849) - Fixed issue that affected Advisors, where Actions and Jobs would always be executed on the primary node in a MongoDB cluster. 
+- [PMM-11934](https://jira.percona.com/browse/PMM-11934) - Product tour is now correctly displaying the Advisors information.
+- [PMM-11960](https://jira.percona.com/browse/PMM-11960) - UI upgrade to 2.37.0 is failed
```

Comment:
I think we can remove it since it was a defect, not a bug

Reply:
Sure